src/box/lua/config/annotations.lua

```diff
@@ -0,0 +1,2162 @@
+    A directory where write-ahead log (`.xlog`) files are stored. A relative
+    path in this option is interpreted as relative to `process.work_dir`.
+    By default, WAL files and snapshots are stored in the same directory.
+    However, you can set different values for the `wal.dir` and `snapshot.dir`
+    options to store them on different physical disks for performance matters.
+]])
+
+M['wal.dir_rescan_delay'] = toline([[
+    The time interval in seconds between periodic scans of the write-ahead-log
+    file directory, when checking for changes to write-ahead-log files for the
+    sake of replication or hot standby.
+]])
+
+M['wal.ext'] = toline([[
+    This section describes options related to WAL extensions.
+]])
+
+M['wal.ext.new'] = toline([[
+    Enable storing a new tuple for each CRUD operation performed. The option
+    is in effect for all spaces. To adjust the option for specific spaces,
+    use the `wal.ext.spaces` option.
+]])
+
+M['wal.ext.old'] = toline([[
+    Enable storing an old tuple for each CRUD operation performed. The option
+    is in effect for all spaces. To adjust the option for specific spaces,
+    use the `wal.ext.spaces` option.
+]])
+
+M['wal.ext.spaces'] = toline([[
+    Enable or disable storing an old and new tuple in the WAL record for a
+    given space explicitly. The configuration for specific spaces has
+    priority over the configuration in the `wal.ext.new` and `wal.ext.old`
+    options. The option is a key-value pair: The key is a space name
+    (string). The value is a table that includes two optional boolean
+    options: old and new. The format and the default value of these options
+    are described in `wal.ext.old` and `wal.ext.new.`
+]])
+
+M['wal.ext.spaces.*'] = 'Space name.'
+
+M['wal.ext.spaces.*.new'] = M['wal.ext.new']
+
+M['wal.ext.spaces.*.old'] = M['wal.ext.old']
+
+M['wal.max_size'] = toline([[
+    The maximum number of bytes in a single write-ahead log file. When a
+    request would cause an `.xlog` file to become larger than `wal.max_size`,
+    Tarantool creates a new WAL file.
+]])
+
+M['wal.mode'] = toline([[
+    Specify fiber-WAL-disk synchronization mode as: none: write-ahead log is
+    not maintained. A node with wal.mode set to none can't be a replication
+    master. write: fibers wait for their data to be written to the
+    write-ahead log (no fsync(2)). fsync: fibers wait for their data,
+    fsync(2) follows each write(2).
+
+]])
+
+M['wal.queue_max_size'] = toline([[
+    The size of the queue in bytes used by a replica to submit new transactions
+    to a write-ahead log (WAL). This option helps limit the rate at which a
+    replica submits transactions to the WAL. Limiting the queue size might be
+    useful when a replica is trying to sync with a master and reads new
+    transactions faster than writing them to the WAL.
+]])
+
+M['wal.retention_period'] = toline([[
+    (EE) The delay in seconds used to prevent the Tarantool garbage collector
```

Comment:
I see no consistency in these `EE` marks. For example, `wal.ext` has no it.

We have `enterprise_edition` annotation in the instance config and can add a remark automatically. It is used, for example, in the `tarantool --help-env-list` table to fill the `AVAILABILITY` column.

Or just drop them all for now. If we add them for some EE options and don't do for some others, it feels confusing.

Reply:
`(EE)` prefix removed.